### PR TITLE
Add clients creator

### PR DESCRIPTION
**Pull request by niksfred**
Ivan's userId (3) hardcoded to have access to all users emails when assigning clients

### CODE REVIEW

File: src/components/AddClient/index.js [hunk 1]
```diff
@@ -26,15 +30,14 @@ export function AddClient({ setIsFormShowing, clientsData, setClientsData }) {
   );
 
   const handleSubmit = async (values) => {
-    const token = window.localStorage.getItem("userToken");
-    const { userId } = parseJwt(token);
-    const newClientData = mapValues(values, userId);
-
+    const newClientData = mapValues(values);
```
Comment (Galabov93) [inline src/components/AddClient/index.js:33]: Please revert the old functionality for non-Ivan emails, meaning:
The Broker dropdown should be visible only for Ivan(and Dani) actually and the rest should not be seeing the dropdown, but instead automatically assign their userID.

File: src/components/AddClient/index.js [hunk 2]
```diff
@@ -54,6 +57,7 @@ export function AddClient({ setIsFormShowing, clientsData, setClientsData }) {
                 realEstatesNeighborhood: [],
                 budget: "",
                 additionalNotes: "",
+                userId: "",
```
Comment (Galabov93) [inline src/components/AddClient/index.js:60]: You could create a function for getting the usedId. 
```suggestion
                userId: getAdminUserId(),
```

File: src/components/AddClient/index.js [hunk 3]
```diff
@@ -126,6 +130,22 @@ export function AddClient({ setIsFormShowing, clientsData, setClientsData }) {
                     handleBlur={handleBlur}
                     errors={errors}
                   />
+                  <FormMultipleSelect
```
Comment (Galabov93) [inline src/components/AddClient/index.js:133]: Create a bool value that shows if the user is admin or not and show the dropdown based on it
```suggestion
                  {isAdminUser && <FormMultipleSelect
```

**Review by Galabov93 — CHANGES_REQUESTED** [review]
@niksfred Nice work. The functionality is working.
There is a need to tweak the functionality a bit so it is more user-friendly.

**Comment by niksfred** [comment]
Couldn't set directly userID in the initial value for userId, so the situation is handled by mappedValues.

**Pull request merged**